.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,7 @@
 blank_issues_enabled: false
 contact_links:
   - name: Bug Reports
-    url: https://jira.mongodb.org/projects/COMPASS/issues/
+    url: https://jira.mongodb.org/
```

Comment:
qq: how would we ensure users create a ticket in the correct project?

Reply:
That's a good point. I don't think there's anything we can do in terms of the link, but I did add some additional helper text to specify the COMPASS project.